rpm/product.spec

```diff
@@ -190,7 +190,8 @@ if [ -L "/home/$genesis_user/run" ]; then
     unlink "/home/$genesis_user/run" || exit 1
 fi
 if [ -d "/home/$genesis_user/run" ]; then
-    rm -rf "/home/$genesisUser/run"
+    rm -rf "/home/$genesis_user/run"
+    echo "data removed run folder" 2>&1 | tee -a "$LOG"
```

Comment:
Can we change this to "detected run folder as a directory instead of a symlink, this folder has been deleted" and add the echo to the other file?

Reply:
echo added in both the files